### ISSUE TITLE
Is it possible to reset/refresh cache on demand?

### ISSUE THREAD

**Issue by EthemAcar-Dev** [dj-nitehawk/FastEndpoints#74]
Is it possible to reset caching via endpoint?

If i have 3 endpoints:
2 endpoints with 4 hour cache setup.
1 endpoint called /refreshcache (reset)
Is it possible to reset the other 2 endpoints cache from the refreshcache endpoint?

**Comment by dj-nitehawk** [comment]
the response caching middleware in asp.net doesn't actually cache anything in memory. 
all it does is set cache headers in the response so that downstream proxy/cache servers and clients/browsers can cache stuff according to those header. you can read more about it at the following link: https://docs.microsoft.com/en-us/aspnet/core/performance/caching/response?view=aspnetcore-6.0

so... as far as the endpoint is concerned, there is no cache for it to reset :-)

in-memory caching is a totally different beast and this library has nothing specific to do with that.
https://docs.microsoft.com/en-us/aspnet/core/performance/caching/memory?view=aspnetcore-6.0

basically, if you want an in-memory cache, you just register it in DI and use whereever you like.
